content/03.results.md

```diff
@@ -119,37 +119,36 @@ Table: **Molecular subtypes generated through the OpenPBTA project.** Listed are
 
 <!-- TODO: Update group-specific count numbers below with figure update -->
 #### Low-grade astrocytic tumors
-**Figure {@fig:Fig2}A** depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, 140/227) of these tumors harbored a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver [@doi:10.1186/s40478-020-00902-z]. <!--SAMPLECOUNT-->
+In **Figure {@fig:Fig2}A**, we show that as expected, the majority (62%, 140/227) of LGGs harbored a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver [@doi:10.1186/s40478-020-00902-z]. <!--SAMPLECOUNT-->
 We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011].
 Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signaling pathway in LGATs (**Figure {@fig:Fig5}B**).
 
 #### Embryonal tumors
-**Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. <!--SAMPLECOUNT-->
-The majority (N = 95) of samples were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see **Molecular Subtyping of CNS Tumors**), as identified by subtype-specific canonical mutations.
-We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
+ <!--SAMPLECOUNT-->
+The majority (N = 95) of samples were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see **Molecular Subtyping of CNS Tumors**), as identified by subtype-specific canonical mutations (**Figure {@fig:Fig2}B**).
+We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as other embryonal tumors) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
 
 #### Diffuse astrocytic and oligodendroglial tumors
```

Comment:
I think these headers should match the figure?
```suggestion
#### High-grade gliomas
```

Reply:
oh, I missed that these changed - yes!